config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Policy.java

```diff
@@ -7,11 +7,16 @@
 package com.ca.apim.gateway.cagatewayconfig.beans;
 
 import com.ca.apim.gateway.cagatewayconfig.config.loader.ConfigLoadException;
+import com.ca.apim.gateway.cagatewayconfig.util.IdGenerator;
+import com.ca.apim.gateway.cagatewayconfig.util.file.DocumentFileUtils;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.ca.apim.gateway.cagatewayconfig.util.file.DocumentFileUtils'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=135&issues=AWcuGG63kEeD8QRk-1yQ&open=AWcuGG63kEeD8QRk-1yQ)

[ISSUE_KEY]: AWcuGG63kEeD8QRk-1yQ